### PR TITLE
 Corrected valid methods in walletUtils.

### PR DESCRIPTION
**Pull request by nogang**
## Proposed changes

- The isValidPrivateKey and isValidAddress logics are incorrect and fixed.

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-java/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-java)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- #19 

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: core/src/main/java/com/klaytn/caver/wallet/KlayWalletUtils.java [hunk 1]
```diff
@@ -188,7 +183,7 @@ public static String getMainnetKeyDirectory() {
 
     public static boolean isValidPrivateKey(String privateKey) {
         String cleanPrivateKey = Numeric.cleanHexPrefix(privateKey);
-        return cleanPrivateKey.length() == PRIVATE_KEY_LENGTH_IN_HEX;
+        return (cleanPrivateKey.length() <= PRIVATE_KEY_HEX_SIZE && cleanPrivateKey.matches("^[0-9A-Fa-f]+$"));
```
Comment (kjhman21) [inline core/src/main/java/com/klaytn/caver/wallet/KlayWalletUtils.java]: Do we allow shorter size of a private key?

Comment (nogang) [reply core/src/main/java/com/klaytn/caver/wallet/KlayWalletUtils.java]: The string for the private key is 64 characters or less unless padded with zeros to match the digits. In addition, it handles the case of receiving private key less than 64 characters in carver-java.

File: core/src/test/java/com/klaytn/caver/feature/KlayWalletUtilsTest.java [hunk 2]
```diff
@@ -20,9 +20,11 @@
 import com.klaytn.caver.wallet.KlayWalletUtils;
 import org.junit.Ignore;
 import org.junit.Test;
+import org.web3j.crypto.Keys;
 import org.web3j.utils.Numeric;
 
 import java.io.File;
+import java.util.Random;
```
Comment (kjhman21) [inline core/src/test/java/com/klaytn/caver/feature/KlayWalletUtilsTest.java]: Why this package is imported? I don't see the usage.

Comment (nogang) [reply core/src/test/java/com/klaytn/caver/feature/KlayWalletUtilsTest.java]: It is obviously a mistake. I used it to make the address random, but the code didn't need to change.

Comment (nogang) [inline core/src/test/java/com/klaytn/caver/feature/KlayWalletUtilsTest.java]:
```suggestion
```

File: core/src/main/java/com/klaytn/caver/wallet/KlayWalletUtils.java [hunk 3]
```diff
@@ -200,7 +195,7 @@ public static boolean isValidAddress(String input) {
             return false;
         }
 
-        return cleanInput.length() == ADDRESS_LENGTH_IN_HEX;
+        return (cleanInput.length() <= ADDRESS_HEX_SIZE && cleanInput.matches("^[0-9A-Fa-f]+$"));
```
Comment (leejh903) [inline core/src/main/java/com/klaytn/caver/wallet/KlayWalletUtils.java]: Inside of `String.matches()` java use `Pattern` instance which is quite expensive object to create. If you create this `Pattern` instance as 'static' variable and reuse it, you could increase performance better. You can use the regex like below.
```
    private static final Pattern ADDRESS = Pattern.compile("^[0-9A-Fa-f]+$");
    
    static boolean isAddress(String s) {
        return ADDRESS.matcher(s).matches();
    }
```

File: core/src/test/java/com/klaytn/caver/feature/KlayWalletUtilsTest.java [hunk 4]
```diff
@@ -99,4 +100,21 @@ public void testLoadKlaytnWalletKeyNormal() {
         assertEquals("0xee135d0b57c7ff81b198763cfd7c43f03a5f7622", credentials2.getAddress());
     }
 
+    @Test
+    public void testValidPrivateKey() throws Exception {
+        String longPrivateKey = Keys.createEcKeyPair().getPrivateKey().toString(16) + "00000000000000000000000000000000000000000000000000000000000000000";
+        assertEquals("longPrivateKey: " + longPrivateKey, false, KlayWalletUtils.isValidPrivateKey(longPrivateKey));
```
Comment (leejh903) [inline core/src/test/java/com/klaytn/caver/feature/KlayWalletUtilsTest.java]: How about this assert expression like below. I think it's more easy to read code.
```suggestion
        assertFalse("longPrivateKey: " + longPrivateKey, KlayWalletUtils.isValidPrivateKey(longPrivateKey));
```

**Pull request merged**